src/nvim/syntax.c

```diff
@@ -3434,9 +3432,7 @@ static void syn_cmd_reset(exarg_T *eap, int syncing)
 {
   eap->nextcmd = check_nextcmd(eap->arg);
   if (!eap->skip) {
-    set_internal_string_var("syntax_cmd", (char_u *)"reset");
-    do_cmdline_cmd("runtime! syntax/syncolor.vim");
-    do_unlet(S_LEN("g:syntax_cmd"), true);
+    init_highlight(true, true);
```

Comment:
this alters the behavior of `syn reset`. This _should_ not be a big deal, and is consistet with the plan that we are staking out, but I am making a note in case it causes an issue (spacebar heating or not) down the line.